source/frontend/src/features/mapSideBar/acquisition/tabs/compensation/detail/CompensationRequisitionDetailContainer.tsx

```diff
@@ -91,11 +91,10 @@ export const CompensationRequisitionDetailContainer: React.FunctionComponent<
   return compensation ? (
     <View
       loading={loading || loadingPerson || loadingOrganization}
+      acquisitionFile={acquisitionFile}
       compensation={compensation}
       compensationContactPerson={payeePerson}
       compensationContactOrganization={payeeOrganization}
-      acqFileProject={acquisitionFile?.project}
```

Comment:
huh, I wonder why these were split out like this?

Reply:
note sure, but now is more cohesive